frontend/src/components/poll/form/index.js

```diff
@@ -66,6 +66,19 @@ export default class extends Form {
 
   onCancel = () => {
     let cancel = false
+    const pollData = this.state
+
+    // Nothing added to the poll so no changes to discard
+    const nothingAddedToPoll =
+      pollData.question === "" &&
```

Comment:
Why use `pollData` instead of `this.state...` like other places do? TBH I like that approach better.

Reply:
True that. Not need to overcomplicate with another variable. Will remove.